### ISSUE TITLE
Forced encounters in RP are broken after the changes for LoadGame_hook.

### ISSUE THREAD

**Issue by NovaRain** [phobos2077/sfall#117]
"Enclave vertibird assault" and "post-ending easter egg" encounters in RP can not be triggered with sfall 3.8.5 or 4.0 after commit c2e6d743a393c3636abbdcc821fc6ba43e0b5c2e. With old LoadGame_hook and with EndLoadHook commented out, both encounters work as intended.

**Comment by NovaRain** [comment]
The script source of the two forced encounters:
```
// Enclave vertibird assault - glgcencl.ssl
procedure start;

procedure start begin
   if (metarule(22,0)) then return;
   if (game_loaded) then begin
      set_global_script_repeat(60);
      set_global_script_type(2);
   end else begin
      if (in_world_map) then begin
         if (global_var(730) == 1) then begin
            force_encounter(161);
         end
      end
   end
end
```
```
// Post-ending easter egg - gleaster.ssl
procedure start;

procedure start begin
   if (metarule(22,0)) then return;
   if (game_loaded) then begin
      set_global_script_repeat(60);
      set_global_script_type(2);
   end else begin
      if (in_world_map) then begin
         if ( (global_var(494) == 8) and (global_var(772) == 0) ) then begin
            set_global_var(772,1);
            force_encounter(166);
         end
      end
   end
end
```

At first I thought force_encounter() is broken, but with my test script I can trigger the forced encounter in world map:
```
procedure start;
#include ".\HEADERS\DEFINE.H"
#include ".\headers\sfall.h"
#include ".\headers\dik.h"

procedure start begin
   if game_loaded then begin
      register_hook(HOOK_KEYPRESS);
   end else begin
      variable
         event := get_sfall_arg,
         keyDX := get_sfall_arg;

      if (event == 1) and (keyDX == DIK_V) then begin
         if (in_world_map) then begin
            if (global_var(730) == 1) then begin
               force_encounter(161);
            end
         end
      end
   end
end
```

**Comment by NovaRain** [comment]
OK, I think I've fixed the bug in my commits on 3.8 and 4.0 branches. Might not be the best solution, but at least the bug in #117 is fixed and the graphic flip issue in #94 is much less noticeable now (thank @Lexx2k and @burner1024 for testing).
